umn/source/faqs/general_issues/how_do_i_upload_data_to_obs.rst

```diff
@@ -5,4 +5,4 @@
 How Do I Upload Data to OBS?
 ============================
 
-Before using ModelArts to develop AI models, data needs to be uploaded to an OBS bucket. You can log in to the OBS console to create an OBS bucket, create a folder, and upload data. For details about how to upload data, see *Object Storage Service Getting Started*.
+Before using ModelArts to develop AI models, data needs to be uploaded to an OBS bucket. You can log in to the OBS console to create an OBS bucket, create a folder, and upload data. For details about how to upload data, see .
```

Comment:
a space before final dot - bad but comes from original. This in turn is a bigger issue that overall link is not still not working `...to upload data, see <em id="modelarts_05_0013__en-us_topic_0000001096750259_en-us_topic_0285164815_en-us_topic_0143299496_i2576427123812"></em>...` - this can't work